app/views/cancelCase/CancelCaseDisclosureReferenceNumberView.scala.html

```diff
@@ -41,20 +41,19 @@
                 classes = Some("govuk-!-width-one-half")
             )
 
-        @button("common.button.continue")
```

Comment:
this originally wasn't part of the story, but the cancel case reference number view needed to be updated too (this progressive disclosure was not placed correctly)